doc/Overview/Mvux/Advanced/Pagination.md

```diff
@@ -0,0 +1,395 @@
+        DataFetchSize="20" Header="Next page auto-loading when on available capacity or scrolled"/>
+
+</Page>
+```
+
+In addition, make sure the `DataContext` of the `Page` is set to the generated bindable model called `BindablePeopleModel` (*MainPage.xaml.cs*):
+
+```csharp
+public MainPage()
+{
+    this.InitializeComponent();
+
+    this.DataContext = new BindablePeopleModel(new PeopleService());
+}
+```
+
+> [!TIP]  
+> You can inspect the generated code by either placing the cursor on the word `BindablePeopleModel` and hitting <kbd>F12</kbd>, see other ways to inspect the generated code [here](xref:Overview.Mvux.Advanced.InspectGeneratedCode).
+
+As you can see, there's nothing special in the XAML code as MVUX is taking advantage of the tools already implemented with the `ListView`.
+
+- When the page load, the first 20 items are loaded (after a delay of one second - as simulated in the service).
+- If there's available space in the `ListView`, the next batch of 20 items will be requested from the Feed and loaded from the service thereafter.
+- When the user scroll down and hits the bottom of the `ListView`, the next 20 items will be requested.
+- This behavior will follow as the user scrolls down and chases the service for more data until all items have been loaded.
+
+Here's what the app renders like:
+
+![](../Assets/PaginationIncrementalLoading.gif)
+
+> [!TIP]  
+> You can use these members of the `ListView` to control data loading: [`DataFetchSize`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.datafetchsize), [`IncrementalLoadingThreshold`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.incrementalloadingthreshold), and [`IncrementalLoadingTrigger`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.incrementalloadingthreshold).
+
+> [!NOTE]  
+> The source-code for the sample app demonstrated in this section can be found [here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MvuxHowTos/PaginationPeopleApp).
+
+## Offset pagination
+
+Offset pagination is controlled via a State that stores the current page index in the Model, and the List-Feed depending on it, using [the `Select` operator](xref:Overview.Mvux.Feeds#select-or-selectasync).  
+When the user requests a new page, the current page index state is updated, thereby updating the dependent collection List-Feed.
+
+Using the example started in [incremental loading above](#incremental-loading), we'll add another method to the service, which will disclose to the View how many items are there in total. Getting a count of items is more efficient than enumerating all entries. This is necessary to identify the total number of pages we have.
+
+### Service
+
+Here's the method added to the service (*PeopleService.cs*):
+
+```csharp
+public async ValueTask<int> GetPageCount(uint pageSize, CancellationToken ct) =>
```

Comment:
This isn't a great example. Most services that serve paged content will return total as part of the paged result, so you don't need to make a separate call to get the page/item count

Reply:
1. We want to know the total number of pages before we start to determine if the "next" button should be available at all
2. We're not adding any new data in our example, so the count is static throughout the app, so why retrieve the count on each pagination request?

Reply:
I'm just saying that returning total number of items in the first request is more common than having a service call that returns the number of pages/items, so I'd prefer to see that as the example
In fact, I've never seen a case where I've had to request the number of items/pages and then make a request for the items

Reply:
how would i know if there is a another page and set the "next" button enabled state accordingly?
we need to have a feed that keeps this info

Reply:
I've redone the pagination page. This method is now only used in "manual pagination" section.